pkgs/development/compilers/llvm/git/llvm/default.nix

```diff
@@ -17,6 +17,7 @@
 , version
 , release_version
 , zlib
+, enableZstd ? true zstd
```

Comment:
This is not valid syntax.

But do we even need an option?  Is there any indication anybody would want to disable it?

Reply:
Also, dependencies like this can often be removed if needed via `.override { zstd = null; }` which has the effect of making `"${zstd}"` expand to an empty string (and therefore disappear from the build system's perspective).